boards/arm/nucleo_h745zi_q/nucleo_h745zi_q_m4_defconfig

```diff
@@ -0,0 +1,26 @@
+# SPDX-License-Identifier: Apache-2.0
+
+CONFIG_SOC_SERIES_STM32H7X=y
+CONFIG_SOC_STM32H745XX=y
+CONFIG_SYS_CLOCK_HW_CYCLES_PER_SEC=400000000
```

Comment:
what about PLL settings to make this value true?

Reply:
Changed to 480MHz and tested with the board.